preprocess_observation_smogon.py

```diff
@@ -32,8 +32,19 @@ def preprocess_observation(I):
                         retval.append([newIndex-1, 0])
                 retval.append([index-1, 1])
                 index = p1a_indices[name]
-                health = int(temp[4].split('/')[0])
-                retval.append([809*2 + index, health])
+                condition = temp[4].split('/')[0].split(' ')
```

Comment:
Again, here we have code replication that suggests a refactor would be helpful.

Reply:
Yes we do